nsv13/code/modules/mob/living/carbon/human/species_types/syndicate_knpc.dm

```diff
@@ -1,8 +1,36 @@
-//Syndie boarders and the gang.
-/mob/living/carbon/human/ai_boarder/commando
-	outfit = /datum/outfit/syndicate/odst/shotgun
+//Syndie boarders
+/mob/living/carbon/human/ai_boarder/syndicate //PARENT - DO NOT USE
+	faction = list("Syndicate")
+	taunts = list(
+		"DEATH TO NANOTRASEN!!",
+		"FOR ABASSI!",
+		"Eat this!",
+		"Weakling!",
+		"You're going home in a coffin kid!",
+		"Engaging the enemy!"
+	)
+	call_lines = list(
+		"Enemy spotted! Need backup in",
+		"OPFOR sighted in",
```

Comment:
Consider adding a format string here like “backup to %LOC%”

Reply:
I currently have a tag that either just takes the call_lines list as is, or adds a %LOC% at the end.